### ISSUE TITLE
Spaces in filename breaks Homebrew

### ISSUE THREAD

**Issue by ringods** [pulumi/homebrew-tap#12]
### What happened?

`pulumi-bot` added spaces to the file name of the archive URL for the last release of `cf2pulumi`:

https://github.com/pulumi/homebrew-tap/blob/b9784efab0f5f7ffc352219296d25f91060c8b77/Formula/cf2pulumi.rb#L13

Commit: https://github.com/pulumi/homebrew-tap/commit/f03fb36f60d6545502404559e552b577fdd929dd

This breaks Homebrew:

```
brew cleanup
Removing: /Users/ringods/Library/Caches/Homebrew/pre-commit_bottle_manifest--3.2.2... (18.8KB)
Removing: /Users/ringods/Library/Caches/Homebrew/azure-cli_bottle_manifest--2.47.0... (20.7KB)
Error: invalid attribute for formula 'pulumi/tap/cf2pulumi': url ("https://github.com/pulumi/pulumi-aws-native/releases/download/v0.71.0/cf2pulumi- v0.71.0- darwin-x64.tar.gz")
Please report this issue:
  https://docs.brew.sh/Troubleshooting
/usr/local/Homebrew/Library/Homebrew/formula.rb:314:in `validate_attributes!'
/usr/local/Homebrew/Library/Homebrew/formula.rb:244:in `initialize'
/usr/local/Homebrew/Library/Homebrew/formulary.rb:484:in `new'
/usr/local/Homebrew/Library/Homebrew/formulary.rb:484:in `get_formula'
/usr/local/Homebrew/Library/Homebrew/formulary.rb:644:in `get_formula'
/usr/local/Homebrew/Library/Homebrew/formulary.rb:761:in `factory'
/usr/local/Homebrew/Library/Homebrew/formulary.rb:852:in `from_keg'
/usr/local/Homebrew/Library/Homebrew/formulary.rb:803:in `from_rack'
/usr/local/Homebrew/Library/Homebrew/cleanup.rb:130:in `stale_formula?'
/usr/local/Homebrew/Library/Homebrew/cleanup.rb:67:in `stale?'
/usr/local/Homebrew/Library/Homebrew/cleanup.rb:429:in `block in cleanup_cache'
/usr/local/Homebrew/Library/Homebrew/cleanup.rb:411:in `each'
/usr/local/Homebrew/Library/Homebrew/cleanup.rb:411:in `cleanup_cache'
/usr/local/Homebrew/Library/Homebrew/cleanup.rb:275:in `clean!'
/usr/local/Homebrew/Library/Homebrew/cmd/cleanup.rb:56:in `cleanup'
/usr/local/Homebrew/Library/Homebrew/brew.rb:94:in `<main>'
```

### Expected Behavior

Without the spaces, Homebrew would work correctly again

### Steps to reproduce

```
$ brew tap pulumi/tap
$ brew install pulumi/tap/cf2pulumi
```

### Output of `pulumi about`

N/A

### Contributing

Vote on this issue by adding a 👍 reaction. 
To contribute a fix for this issue, leave a comment (and link to your pull request, if you've opened one already).

**Comment by mjeffryes** [comment]
I bet it's this change: https://github.com/pulumi/pulumi-aws-native/commit/fb187a509df32e17baa255f9108c31cf1cf6334d

**Comment by mjeffryes** [comment]
Should be fixed in release 0.72.0: https://github.com/pulumi/pulumi-aws-native/releases/tag/v0.72.0
